src/view/edit-point.js

```diff
@@ -250,19 +264,50 @@ class EditPoint extends SmartView {
     this.getElement().querySelector('form').addEventListener('submit', this._formSubmitHandler);
   }
 
+  _priceChangeHandler(evt) {
+    const newPrice = +evt.currentTarget.value;
```

Comment:
```suggestion
    const newPrice = evt.currentTarget.valueAsNumber;
```
не все об этом знают, а ты знай 😉